python/hiro-client.py

```diff
@@ -42,16 +43,25 @@ def __init__(self, username, password,
 
     @backoff.on_exception(*BACKOFF_ARGS, **BACKOFF_KWARGS)
     def get(self, url, token=None):
+        url = url.replace('"','') # temporary workaround.
+
         if token is None:
             token, _token_expired = self.get_token()
         headers = self._headers
         if token is not None:
             headers['Authorization'] = "Bearer " + token
         res = requests.get(url, headers=headers, verify=False)
-        return self._parse_response(res)
```

Comment:
Why reverting the handling of the requests response here, from a single method `_parse_response` to individual code blocks for get, post, delete? It looks to me like the handling block ( `if res.status_code == 401 ...return res.text`) is the same for the three access modes get/post/delete, so we have code duplication here.